Add makeflyeasy believer fixedwing model

Jaeyoung-Lim opened the pull request:
**Problem Description**
This commit adds the [makeflyeasy Believer](https://ardupilot.org/plane/docs/airframe-believer.html) fixedwing vehicle as a sdf model for PX4 SITL.

This model is used for testing A-tail configuration as well as a dual-motor fixedwing aircraft

**Testing**
```
make px4_sitl gazebo_believer
```
![image](https://user-images.githubusercontent.com/5248102/118088555-85d31880-b3c7-11eb-86ed-523c11974714.png)

TSC21 commented on models/believer/model.config:
```diff
@@ -0,0 +1,14 @@
+<model>
+  <name>Believer</name>
+  <version>1.0</version>
+  <sdf version='1.5'>believer.sdf</sdf>
```
any reason to not use a higher version?

TSC21 commented on models/believer/believer.sdf:
```diff
@@ -0,0 +1,693 @@
+<sdf version='1.6'>
```
Not matching the one on the config file.

Jaeyoung-Lim replied:
Fixed!

TSC21 commented on models/believer/believer.sdf:
```diff
@@ -0,0 +1,693 @@
+    <link name='base_link'>
+      <pose>0 0 0 0 -0 0</pose>
+      <inertial>
+        <pose>0.15 0 0.1 0 -0 0</pose>
```
Maybe for cleanup remove all the `-0`?

Jaeyoung-Lim replied:
Cleaned up!

TSC21 commented on models/believer/believer.sdf:
```diff
@@ -0,0 +1,693 @@
+        <pose>0 0 -0.07 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>1.088 2.591 0.103</size>
```
wondering if you could use the a downsized STL as the collision model.

Jaeyoung-Lim replied:
We could, but I am not sure what value this would bring. Currently the box allows you to smoothly takeoff from the ground :)

TSC21 commented on models/believer/believer.sdf:
```diff
@@ -0,0 +1,693 @@
+        <surface>
+          <contact>
+            <ode>
+              <max_vel>10</max_vel>
```
any reason for this?

Jaeyoung-Lim replied:
This allows the model to slide on the floor for takeoff

TSC21 commented on models/believer/believer.sdf:
```diff
@@ -0,0 +1,693 @@
+      <velocity_decay/>
+      <self_collide>0</self_collide>
+    </link>
+    <link name='believer/imu_link'>
```
do we need the `believer` prefix?

Jaeyoung-Lim committed:
Add Believer fixedwing model

This commit adds the makeflyeasy Believer fixedwing vehicle as a sdf model for PX4 SITL.

This model is used for testing A-tail configuration as well as a dual-motor fixedwing aircraft
